### PR TITLE
Added a panel containing a list of pokemon currently on the map.

### PR DESCRIPTION
**Pull request by Jerrkawz**
Sometimes I find it easier to scan a list than to look over a map so I added a list of the currently displayed pokemon.
## Description
I added a second panel next to the filter one that behaves the same way. I chose a panel because I wanted a small list that could be easily scanned without having to swap back and forth to a full grid view for example. I tried to code it exactly as the existing filter panel as much as possible. Eventually I want to add support for sorting in this list but the initial drop doesn't sort the list at all.

## Motivation and Context
It's an additional mode of displaying pokemon that I think others will find useful. It doesn't take away from anything that already exists. It only adds extra functionality that the user can choose to ignore if they want.

## How Has This Been Tested?
Careful coding and manual testing.

## Screenshots (if appropriate):
image

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by Ayysir** [comment]
I like this

File: static/map.js [hunk 1]
```diff
@@ -115,7 +115,7 @@ function initMap() {
     google.maps.event.addListenerOnce(map, 'idle', function(){
         updateMap();
     });
-    
+
```
Comment (Chlodochar) [inline static/map.js]: This is an unrelated change. Please remove it from the PR.

File: static/map.js [hunk 2]
```diff
@@ -914,7 +932,7 @@ $(function () {
             map_pokemons = {}
         }
     });
-    
+
```
Comment (Chlodochar) [inline static/map.js]: This is an unrelated change. Please remove it from the PR.

File: static/sass/layout/_list.scss [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+     }
+
+     .hide_pokemon  {
+       float: right;
+        > a {
+         color: #b20a31;
+         text-decoration: none;
+       }
+     }
+ 	}
```
Comment (Chlodochar) [inline static/sass/layout/_list.scss]: There are mixed tabs and spaces in here. Please use spaces and fix the indentation.

File: static/js/app.js [hunk 4]
```diff
@@ -62,10 +80,18 @@
 	$navClose.tabIndex = 0;
 	$nav.appendChild($navClose);
 
+	$listClose = document.createElement('a');
```
Comment (Chlodochar) [inline static/js/app.js]: Tabs followed by spaces. Please use tabs here (I know, it's already a mess ;)).

**Comment by sparkycus123** [comment]
hi sir, can you guide me on how to apply these enhancements on my map please

**Comment by Jerrkawz** [comment]
Rebasing, the pull request is messed up atm, give me a few minutes.

**Comment by Jerrkawz** [comment]
@Chlodochar Alright sorry about the number of commits but the branch diff should be correct now. I did a rebase and by the time i merged its already out of date again haha, it may be easier for you to merge but if not just let me know.

**Comment by Jerrkawz** [comment]
@sparkycus123 You can either clone my fork, which is mostly up to date:

1. git clone https://github.com/Jerrkawz/PokemonGo-Map.git
2. git checkout develop

and build/run it the same way or you can wait until (hopefully) the changes are merged into this repo. Thanks for trying it out!

**Comment by andrewda** [comment]
+1 but needs a rebase

**Comment by Jerrkawz** [comment]
@andrewda Rebased and squashed to a single commit.

**Comment by kamykaze** [comment]
I tried this out and really like it. Here are a couple suggestions:

1. Add click event to the name of the pokemon so it centers the map around the pokemon
2. Sort by distance or rarity (would need to merge with #1899 or something that gives you the rarity info

For the first request, I was able to do it by adding the following code (see diff txt), but I'm not sure if there's a better way to go about this.

[center_map.txt](https://github.com/AHAAAAAAA/PokemonGo-Map/files/382609/center_map.txt)

**Comment by Jerrkawz** [comment]
@kamykaze Great suggestions. I was thinking about making the entire card clickable and then it centers and opens the info box on the map. I wanted to get the original PR in before I started adding more functionality though because merging has become my nightmare.

**Comment by kamykaze** [comment]
@Jerrkawz I agree. I was using this list for a couple days but merging the latest develop branch causes conflicts each time.

I hope this gets merged  soon. It's a great feature.

**Comment by Jerrkawz** [comment]
Yeah merging is no fun. And development on this is moving so fast if i rebase its outdated in 30 minutes. I'll have to work closely with someone and rebase right before the PR is merged.

**Comment by Jerrkawz** [comment]
@andrewda Just rebased to pick up the latest changes. If you guys want to merge it now would be the time

**Comment by Nicknack1234** [comment]
Is this not like what we have in the current map if browse to /mobile it displays a list of visible pokemon with direction and distance

**Comment by AHAAAAAAA** [comment]
Somehow this got buried under the 101 PRs. This looks great and should be merged, it's useful to have even in the full map. Can I ask you for one more rebase?

**Comment by Jerrkawz** [comment]
Hey glad you like it. I'm currently out of the country for another 10 days and I won't be able to get to a computer. @ctsoccer13 has access to my repo perhaps he can rebase for me? Or I can around August 14th. Thanks
